docusaurus/docs/Android/04-compose/03-channel-components/02-channel-list-header.mdx

```diff
@@ -81,35 +79,25 @@ In terms of UI customization, the `ChannelListHeader` exposes the following prop
 fun ChannelListHeader(
     modifier: Modifier = Modifier,
     title: String = "",
-    currentUser: User? = ChatClient.instance().getCurrentUser(),
+    currentUser: User? = null,
     connectionState = ConnectionState.CONNECTED,
-    leadingContent: @Composable RowScope.() -> Unit = {
-        DefaultChannelListHeaderLeadingContent(currentUser, onAvatarClick) 
-    },
-    centerContent: @Composable RowScope.() -> Unit = {
-        DefaultChannelListHeaderCenterContent(
-            modifier = Modifier.weight(1f),
-            connectionState = connectionState,
-            title = title
-        )
-    },
-    trailingContent: @Composable RowScope.() -> Unit = {
-        DefaultChannelListHeaderTrailingContent(onHeaderActionClick) 
-    },
+    leadingContent: @Composable RowScope.() -> Unit = { ... },
+    centerContent: @Composable RowScope.() -> Unit = { ... },
+    trailingContent: @Composable RowScope.() -> Unit = { ... },
     ... // Action handlers
 )
 ```
 
-* `modifier`: Modifier for the root component styling. You can add things like a background, elevation, padding and more.
+* `modifier`: Modifier for the root component styling. You can add things like background, elevation, padding, and more.
```

Comment:
If I'm not mistaken commas before 'and' in lists are only used when joining two separate clauses.

E.g. "Hey that's an awesome car, and its red!

I'd leave the final decision up to you or @filbabic as this is a minor grammatical annoyance and I'm not 100% sure I'm reading the rules right.

Reply:
We mostly use commas before 'and' if there are more than 3 list items.

Reply:
I wasn't aware of that rule.
Thank you Amit.